package.json

```diff
@@ -95,85 +93,77 @@
     "lint:fix": "eslint . --ext .js,.ts --fix",
     "clean": "rimraf coverage/ commonjs/ dist/ es/ languages/ lib/ typings/ test-jasmine/",
     "compile": "tsc",
-    "compile:tests": "tsc --project tsconfig.test.json",
-    "jasmine:start": "cross-env NODE_ICU_DATA=node_modules/full-icu jasmine --config=jasmine.json",
-    "check:licenses": "license-checker --production --excludePackages=\"hyperformula@0.0.1\" --onlyAllow=\"MIT; Apache-2.0; BSD-3-Clause; BSD-2-Clause; ISC; BSD; Unlicense\"",
+    "check:licenses": "license-checker --production --excludePackages=\"hyperformula@2.7.1\" --onlyAllow=\"MIT; Apache-2.0; BSD-3-Clause; BSD-2-Clause; ISC; BSD; Unlicense\"",
     "tsnode": "ts-node --transpile-only -O {\\\"module\\\":\\\"commonjs\\\"}"
   },
   "dependencies": {
     "chevrotain": "^6.5.0",
     "tiny-emitter": "^2.1.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.8.4",
-    "@babel/plugin-transform-runtime": "^7.8.3",
-    "@babel/preset-env": "^7.8.4",
-    "@babel/register": "^7.9.0",
-    "@babel/runtime": "^7.18.9",
-    "@babel/preset-react": "^7.13.13",
-    "@babel/preset-typescript": "^7.13.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.13.8",
+    "@babel/cli": "^7.25.9",
+    "@babel/core": "^7.26.0",
+    "@babel/plugin-transform-runtime": "^7.25.9",
```

Comment:
The same as for `core-js`? https://github.com/handsontable/hyperformula/blob/feature/issue-1451/babel.config.js#L21

Reply:
Done. Thank you for spotting it ;)